### PR TITLE
Create Stop-SPOTenantRename

### PR DESCRIPTION
**Pull request by WayneEwington**
Documenting the new PowerShell cmdlets to support SPO Tenant Renames.

Please include this PR with the other files:
* Start-SPOTenantRename
* Get-SPOTenantRenameStatus
* Stop-SPOTenantRename

### CODE REVIEW

**Comment by WayneEwington** [comment]
@yogkumgit  How long is it expected for JohanFreelancer9 to do the review as this article needs to be published. (Also note that I can't @mention him.)

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+office-docs-powershell/sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename
+@WayneEwington
+WayneEwington Create Stop-SPOTenantRename
+…
+Latest commit 638e164 1 minute ago
+ History
+ 1 contributor
+95 lines (65 sloc)  2.68 KB
+  
+---
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:35]: I think that when you copied the content all of the above wasn't intended.

**Review by get-itips — CHANGES_REQUESTED** [review]
@WayneEwington please rename the file adding the ".md" extension

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## PARAMETERS
+
+### -Reason
+
+Specifies the reason why the job is being canceled.
+
+```yaml
+Type: String
+Parameter Sets: Default
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:88]:
```suggestion
Parameter Sets: (All)
```

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+Accept wildcard characters: False
+```
+
+### -Comment
+
+Specifies an additional comment why the job is being canceled.
+
+```yaml
+Type: String
+Parameter Sets: Default
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:105]:
```suggestion
Parameter Sets: (All)
```

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+[Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
+
+[Rename your SharePoint domain](https://aka.ms/SPOTenantRename)
+
+[Start-SPOTenantRename](Start-SPOTenantRename.md)
+
+[Get-SPOTenantRenameStatus](Get-TenantRenameStatus.md)
+
+[Get-SPOSiteRenameState](Get-TenantSiteState.md)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:129]: this should be the end of the file, the following text wasn't probably intended to be included.

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+author: WayneEwington
+ms.author: waynewin
+ms.reviewer:
+---
+
+# Stop-SPOTenantRename
+
+## SYNOPSIS
+
+ [!IMPORTANT]
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:51]:
```suggestion
 [!IMPORTANT]
```

Same as https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/11 and https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/12.

**Comment by yogkumgit** [comment]
@WayneEwington Please implement the suggested changes, if you agree. Thanks

**Pull request closed without merging**